mqttx-web/umbrel-app.yml

```diff
@@ -0,0 +1,22 @@
+manifestVersion: 1
+id: mqttx-web
+name: MQTTX Web
+tagline: Your All-in-One MQTT Client Toolbox
+icon: ""
+category: automation
+version: "1.11.0"
+port: 1884
```

Comment:
I'm thinking we should probably reserve this port in case an mqtt broker app comes to the app store. 1884 is a common alternate port for this.

And in this case, the port for the UI here can be any free port. For example `9002` is currently free.

Reply:
fixed to 9010, as 9001 is used by mqtt via websocket, and 9002 may be likley an alternate port for that.

Reply:
Looks like 9010 is sneakily used by MinIO for the API. I have changed to free port 9012.